gammapy/time/lightcurve.py

```diff
@@ -68,3 +69,35 @@ def simulate_example(cls):
         lc['FLUX_ERR'] = Quantity([0.1, 0.4, 0.7, 0.9], 'cm^-2 s^-1')
 
         return lc
+
+    def compute_fvar(self):
+        """Calculate the fractional excess variance (Fvar) of `LightCurve`.
+
```

Comment:
If you have time, you could give a formula or example in the method docstring like here:

http://docs.gammapy.org/en/latest/api/gammapy.stats.significance.html
http://docs.gammapy.org/en/latest/_modules/gammapy/stats/poisson.html#significance